win32ss/user/ntuser/kbdlayout.c

```diff
@@ -702,10 +667,10 @@ HKL APIENTRY
 co_UserActivateKeyboardLayout(
     _Inout_ PKL     pKL,
     _In_    ULONG   uFlags,
-    _Inout_ PWND    pWnd)
+    _Inout_opt_ PWND pWnd)
```

Comment:
This parameter doesn't seem to be used as out. (Neither the pKL one above.)

Reply:
Fixed as `_In_opt_ PWND pWnd`.
`pKL` is `_Inout_` because `pKL->wchDiacritic = UNICODE_NULL;`.